debian/yaru-theme-mate-abandon-Yaru-MATE-light-and-Yaru-MATE-dark.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+yaru_mate_light_name="'Yaru-MATE-light'"
+yaru_mate_dark_name="'Yaru-MATE-dark'"
+yaru_mate_new_name="'Yaru-mate'"
+yaru_mate_dark_new_name="'Yaru-mate-dark'"
+yaru_cursor_name="'Yaru'"
+
+if [ "$(gsettings get org.mate.desktop.interface gtk-theme)" = "$yaru_mate_light_name" ]; then
+    gsettings set org.mate.desktop.interface gtk-theme "$yaru_mate_new_name"
+fi
+
+if [ "$(gsettings get org.mate.desktop.interface gtk-theme)" = "$yaru_mate_dark_name" ]; then
+    gsettings set org.mate.desktop.interface gtk-theme "$yaru_mate_dark_new_name"
+fi
+
+if [ "$(gsettings get org.mate.desktop.interface icon-theme)" = "$yaru_mate_light_name" ] || [ "$(gsettings get org.mate.desktop.interface icon-theme)" = "$yaru_mate_dark_name" ]; then
+    gsettings set org.mate.desktop.interface icon-theme "$yaru_mate_new_name"
+fi
+
+if [ "$(gsettings get org.mate.desktop.peripherals.mouse cursor-theme)" = "$yaru_mate_light_name" ] || [ "$(gsettings get org.mate.desktop.peripherals.mouse cursor-theme)" = "$yaru_mate_dark_name" ]; then
+    gsettings set org.mate.desktop.peripherals.mouse cursor-theme "$yaru_cursor_name"
+fi
```

Comment:
It is normal to migrate a dark variant of the icon and cursor theme? 🤔

Reply:
[`ubuntu-mate-artwork`](https://github.com/ubuntu-mate/ubuntu-mate-artwork) shipped with both `Yaru-MATE-light` and `Yaru-MATE-dark` icon and cursor themes ([_see here in their repo_](https://github.com/ubuntu-mate/ubuntu-mate-artwork/tree/master/usr/share/icons)). 

That's why `icon-theme` and `cursor-theme` should also be migrated.

Reply:
Oh, I didn't know.
Thanks very much!